doc/source/methods/Similarity.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<p>Equations from the first paragraph are not rendering properly.</p><p>Some characters (e.g. epsilon) are not rendering properly.</p><p>I would try to be consistent with the notation. Sometimes you include the parameters as a subscript, otherwise (when defined the kernels) you add them with the input instances (k(x, y, \theta). If you decide to keep it in parenthesis, please include a <code>;</code> to separate the inputs from the parameters. Is a more standard notation. </p><p> </p><p>gradient based -&gt; gradient-based</p><p>maybe consider (for example a classifier) -&gt; (e.g., a classifier) ?</p><p>which reads like -&gt; defined as?</p><p> </p><p>In fact, by substituting this value in the Taylor expansion above we obtain -&gt; By substituting (this -&gt; \delta{\theta}) in the Taylor expansion, we obtain:</p><p> </p><p>That’s it, the dot product kernel --- might get confused with grad-dot. So maybe is worth mentioning  explicitly that this is the asymmetric dot product?</p><p> </p><p>In the definitions of grad-dot and grad-cos I would try to remove the <code>and</code> , display them in two lines, label or index them so we can refer to them later. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>